lib/factorials/factorials.js

```diff
@@ -0,0 +1,12 @@
+const pry = require('pryjs');
+
+function factorial(number) {
+  var factorialNumbers = [];
+  var n = number;
+  while(n > 0) {
+    factorialNumbers.push(n);
+    n --;
+  };
+  return factorialNumbers.reduce((sum, value) => sum * value);
```

Comment:
👍  it seems like this function is doing pretty separate pieces of logic. could extract them out into functions and just call them here?